### PR TITLE
Comments

### PR DESCRIPTION
**Pull request by Heydon**
Drafted comments component. Includes a change to the form validation script, enabling the emission of a custom event to hook into.

### CODE REVIEW

File: src/components/comments.md [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+    <span>Newest first</span>
+  </label>
+  <label>
+    <input name="sort" type="radio" value="timeAsc">
+    <span>Oldest first</span>
+  </label>
+</fieldset>
+```
+
+Screen readers identify both the fieldset (by the `<legend>` group label[^6]) and the individual inputs. When focused is moved onto the `checked` _"Newest first"_ radio, users will hear something similar to _"Newest first, selected radio button, 1 of 2, group, Sort by"_. The order of readout and the precise wording varies between screen readers, browsers, and operating systems.
```
Comment (micmath) [inline src/components/comments.md]: Suggest "When **focus** is moved onto the..."

File: src/components/comments.md [hunk 2]
```diff
@@ -0,0 +1,342 @@
...
+* **`class="gel-form__field-error"`:** The form should use the standard and accessible error messaging mechanism described in [**Form fields**](../form-fields). This error element is associated with the `<textarea>` and populated via the [**Form fields**](../form-fields) implementation's script.
+* **`id="notice"`:** All users should be made aware of the moderation rules before submitting their comment. To make this information available to screen reader users in a timely fashion, it is associated with the submit button using `aria-describedby`[^2]. That is: it will be read out as part of the button's semantic information while the user is focused on it.
+* **`href="#comment"`:** The comment form should be easily accessible at the top of the comments landmark. A link to the _"Add your comment"_ field is also available _after_ the list of comments. This allows a user whose reached the end of the comments to directly access the comment form to add their own.
```
[the remark this answers is not in the document]
Comment (Heydon) [reply src/components/comments.md]: Whoops!

File: src/components/comments.md [hunk 3]
```diff
@@ -0,0 +1,342 @@
...
+
+The comments should be grouped into a list, since lists are identified and their items enumerated in screen reader output. Screen readers also provide list navigation shortcuts, such as the <kbd>i</kbd> key for navigating to the next list item in NVDA[^4].
+
+The following example illustrates a single comment, as a list item:
+
+```html
+<li class="gel-comment" id="lmu4jrrnt" tabindex="-1">
+  <h3>
+    Clive Warren
+    <time datetime="2019-07-10T13:48">on Wed Jul 10 2019, at 13:48</time>
```
[the remark this answers is not in the document]
Comment (Heydon) [reply src/components/comments.md]: @micmath Do you know how to get that from a timestamp using `Date` and, preferably, not a Date parsing library?

**Comment by micmath** [comment]
Thank you.

**Pull request merged**